geode-docs/tools_modules/gfsh/command-pages/connect.html.md.erb

```diff
@@ -175,16 +175,16 @@ gfsh>connect
 ``` pre
 gfsh>connect
 Connecting to Locator at [host=localhost, port=10334] ..
-Connecting to Manager at [host=GemFireStymon, port=1099] ..
-Successfully connected to: [host=GemFireStymon, port=1099]
+Connecting to Manager at [host=GeodeStymon, port=1099] ..
```

Comment:
Lines 178 & 179 are just host names, but as they are the only referece left to "GemFire" I changed it to "Geode"